docker/production.yml

```diff
@@ -227,6 +227,20 @@ services:
       # Disable Elasticsearch routing via Traefik in production (we enable it in development)
       - 'traefik.enable=false'
 
+  portainer:
+    image: portainer/portainer:alpine
+    container_name: 'portainer'
+    command: -H unix:///var/run/docker.sock --admin-password-file '/data/portainer'
```

Comment:
This is probably good, until we can get our auth service set up properly, then maybe we can configure portainer to use our auth service.